plugin/src/main/java/org/opensearch/ml/action/connector/GetConnectorTransportAction.java

```diff
@@ -96,16 +104,11 @@ protected void doExecute(Task task, ActionRequest request, ActionListener<MLConn
                                 XContentParser parser = r.parser().get();
                                 ensureExpectedToken(XContentParser.Token.START_OBJECT, parser.nextToken(), parser);
                                 Connector mlConnector = Connector.createConnector(parser);
+                                if (!TenantAwareHelper
+                                    .validateTenantResource(mlFeatureEnabledSetting, tenantId, mlConnector.getTenantId(), actionListener))
+                                    return;
```

Comment:
Suggest check all places add `{}`
```suggestion
                                    .validateTenantResource(mlFeatureEnabledSetting, tenantId, mlConnector.getTenantId(), actionListener)) {
                                    return;
                                }
```